# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/gnome-terminal/login-shell.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- gnome-terminal-3.58.0/src/org.gnome.Terminal.gschema.xml.vanilla	2025-12-06 19:23:53.321473475 +0200
+++ gnome-terminal-3.58.0/src/org.gnome.Terminal.gschema.xml	2025-12-06 19:24:03.276776729 +0200
@@ -270,7 +270,7 @@
       <description>Possible values are “close” to close the terminal, “restart” to restart the command, and “hold” to keep the terminal open with no command running inside.</description>
     </key>
     <key name="login-shell" type="b">
-      <default>false</default>
+      <default>true</default>
       <summary>Whether to launch the command in the terminal as a login shell</summary>
       <description>If true, the command inside the terminal will be launched as a login shell (argv[0] will have a hyphen in front of it).</description>
     </key>
